include/Client.h

```diff
@@ -79,5 +79,7 @@ DECL_RETRIEVAL_CMD(gets)
   std::vector<unsigned_result_t*> m_outUnsignedResultPtrs;
 };
 
+const char* errCodeToString(err_code_t err);
```

Comment:
maybe move errCodeToString to `Common.h` ?

Reply:
okay